src/main/java/org/broadinstitute/hellbender/dev/pipelines/bqsr/DataflowReadsFilter.java

```diff
@@ -0,0 +1,58 @@
+package org.broadinstitute.hellbender.dev.pipelines.bqsr;
+
+import com.google.cloud.dataflow.sdk.transforms.DoFn;
+import com.google.cloud.dataflow.sdk.transforms.PTransform;
+import com.google.cloud.dataflow.sdk.transforms.ParDo;
+import com.google.cloud.dataflow.sdk.values.PCollection;
+import htsjdk.samtools.SAMFileHeader;
+import org.broadinstitute.hellbender.engine.filters.ReadFilter;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+import org.broadinstitute.hellbender.utils.read.MutableGATKRead;
+
+import java.io.Serializable;
+
+/**
+ * A simple filter for reads on dataflow
+ */
+public final class DataflowReadsFilter extends PTransform<PCollection<MutableGATKRead>, PCollection<MutableGATKRead>> implements Serializable {
+    private static final long serialVersionUID = 1L;
+    
+    private final ReadFilter readFilter;
+    private final SAMFileHeader header;
+
+    /**
+     * ReadsFilter will use the given header to interpret each read, and then let the Read through
+     * if the passed filter accepts it.
+     */
```

Comment:
Ideally, yes, but if we want to adopt that convention we should do so globally across the whole codebase as part of a separate ticket.